<launch>
  <arg name="event_dev" default="/dev/input/gamepads/event_dft" />

  <group ns="teleop">
    <node pkg="sixad_rumble" type="rumble_event_node" name="joy_rumble_node" output="screen">
      <param name="device" value="$(arg event_dev)" />
      <param name="duration_sec" value="3.0" />
      <remap from="~set_rumble" to="/teleop/set_joy_rumble" />
    </node>
  </group>
</launch>
